gradle.properties

```diff
@@ -3,7 +3,6 @@ pluginSinceBuild=241
 pluginUntilBuild=242.*
 
 javaVersion=17
-javaTargetVersion=17
```

Comment:
Can you explain why javaTargetVersion removed?

Reply:
This property was initially added with the intention of specifying different source and target compatibility versions for Java. However, as both the source and target compatibility versions are now set to the same Java version, the property is no longer needed. Therefore, this line has been removed from gradle.properties.

Reply:
Thanks for the clarification @staicy123